plugins/easyimage/plugin.js

```diff
@@ -198,6 +198,8 @@
 			widgetDefinition = CKEDITOR.plugins.imagebase.addFeature( editor, 'link', widgetDefinition );
```

Comment:
After I rebased it onto latest `t/932-2961` branch - that brought Balloon Panel, and it has an issue. I'll add a manual test for that shortly.

Reply:
There's other issue related to blurring:

1. Focus a EI widget with no caption, so that placeholder text appears.
2. Put the focus outside of the editor.

## Expected

Caption is removed/hidden.

## Actual

Caption + placeholder remains visible.

Added manual test for that in [tests\plugins\imagebase\features\manual\captionblur.md](https://github.com/ckeditor/ckeditor-dev/blob/8115f8d/tests/plugins/imagebase/features/manual/captionblur.md#L5) test.